File: iconify_full/android/src/main/java/dev/iconify/full/IconifyFullPlugin.java
package dev.iconify.full;

import androidx.annotation.NonNull;
import io.flutter.embedding.engine.plugins.FlutterPlugin;

/** Build-hook plugin; subset runs via Gradle before Android builds. */
public class IconifyFullPlugin implements FlutterPlugin {
  @Override
  public void onAttachedToEngine(@NonNull FlutterPluginBinding binding) {}

  @Override
  public void onDetachedFromEngine(@NonNull FlutterPluginBinding binding) {}
}

File: iconify_full/android/src/main/kotlin/dev/iconify/full/IconifyFullPlugin.kt
package dev.iconify.full

import io.flutter.embedding.engine.plugins.FlutterPlugin

/** Build-hook plugin; subset runs via Gradle before Android builds. */
class IconifyFullPlugin : FlutterPlugin {
    override fun onAttachedToEngine(binding: FlutterPlugin.FlutterPluginBinding) {}

    override fun onDetachedFromEngine(binding: FlutterPlugin.FlutterPluginBinding) {}
}
